utils/vim/syntax/swift.vim

```diff
@@ -10,10 +10,8 @@ endif
 syn keyword swiftKeyword
       \ associatedtype
       \ break
-      \ case
```

Comment:
The reason to remove `case` and `default` from keywords is that keywords overrule any other syntax item such as region and match.